### ISSUE TITLE
Provide test code (MIDI or Arduino) to test out the LEDs once the assembly is finished

### ISSUE THREAD

**Issue by arthurlutz** [silveirago/traktorino#4]
Once the assembly is finished it would be nice to have a test program to test out all the LEDs, two solutions could be used : 

* some sort of python script or other that talks to midi and turns on and off the LEDs by sending MIDI signales

* an arduino sketch that goes through all the LEDs and turns them on and off and prints out some debug information on the serial port.
